src/palindrom.js

```diff
@@ -11,9 +11,20 @@ if(typeof require !== 'undefined') {
   var JSONPatchOT = require('json-patch-ot');
   var JSONPatchOTAgent = require('json-patch-ot-agent');
   var URL = require('./URL');
+  var axios = require('axios');
+
+  var NodeWebSocket = require('websocket').w3cwebsocket;
+
+  /* this allows us to stub WebSockets */
+  if(!global.WebSocket) { /* we are in production env */
+    var WebSocket = NodeWebSocket;
```

Comment:
Not exactly, because[ imports are hoisted](http://exploringjs.com/es6/ch_modules.html#_imports-are-hoisted), and I prefer my code to look as close as possible to its behaviour. this line
```js
var NodeWebSocket = require('websocket').w3cwebsocket;
```
will be executed even it was inside an if without a `false` condition.

So keeping it above is much more predictable.